integration/spark-common/src/main/scala/org/apache/carbondata/spark/util/CommonUtil.scala

```diff
@@ -169,7 +169,7 @@ object CommonUtil {
       partitionType.get.toUpperCase() match {
         case "HASH" => if (!numPartitions.isDefined) isValid = false
         case "LIST" => if (!listInfo.isDefined) isValid = false
-        case "RANGE" => if (!rangeInfo.isDefined) isValid = false
+        case "RANGE" => isValid = validateRangeInfo(rangeInfo)
```

Comment:
Hi @manishgupta88, the rangeInfo validation should base on column data type, actually the validate function has been developed in my dev branch. Will submit in my next pr. Thanks although. ：）